analyzer/protocol/openvpn/openvpn_zeek.spicy

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
+module OpenVPN_Zeek;
+
+import OpenVPN;
+import zeek;
+
+on OpenVPN::ControlMessage::%done {
```

Comment:
This might be a bit of a side-note - but is this logic really necessary to avoid false positives? We have had stuff like this in other protocols in the past (looking at TLS - where it, admittedly, was much more complex), and it actually, in the end, mostly prevented parsing sessions that used the protocol in slightly unusual ways.

Reply:
Without it this was detecting a lot of things that aren't OpenVPN on 3 big live networks I've tested this logic on (the binpac version, spicy is too new for me to push out like that).  I'm not a fan of detecting things that I'm not very confident are the protocol I'm looking for.

Reply:
Additional note:  the handshake is also captured in dpd.sig.  I'll test removing this logic and rely just on dpd.sig to cut out most of the other protocols and report back.

Reply:
Taking the handshake logic and dpd sigs out isn't that bad of a change on my test networks today.  I'm pushing the code changes to this MR today.

Reply:
I now remember why I put this handshake in.  Some DNS queries without a reply from a server will look like an OpenVPN connection that didn't fully happen.

Reply:
Handshake detection removed, added a check to not conflict with the occasional dns query.  Should be good now.

Reply:
What makes this more interesting is I actually see real openvpn usage on port 53/udp on a real network with this analyzer.  :)